tests/jenkins/lib-testers/TestDownloadFromS3LibTester.groovy

```diff
@@ -0,0 +1,46 @@
+import static org.hamcrest.CoreMatchers.notNullValue
+import static org.hamcrest.MatcherAssert.assertThat
+
+class TestDownloadFromS3LibTester extends LibFunctionTester {
```

Comment:
Let's rename the class to `DownloadFromS3LibTester`

Reply:
This file name is `TestUploadToS3LibTester` so I think these two are similar. https://github.com/opensearch-project/opensearch-build/blob/main/tests/jenkins/lib-testers/TestUploadToS3LibTester.groovy

Reply:
ohh. That's my bad then. I'll rename that file as well. The other 2 are named correctly. Thanks for pointing it out!

Reply:
https://github.com/opensearch-project/opensearch-build/pull/1562